packages/nvidia_nat_autogen/tests/test_callback_handler_integration.py

```
@@ -0,0 +1,560 @@
+        # Subscribe to intermediate step events
+        ctx = Context.get()
+        ctx.intermediate_step_manager.subscribe(event_capturer)
+
+        # Get AutoGen client with invalid key
+        await builder.add_llm("test_llm", invalid_config)
+        client = await builder.get_llm("test_llm", wrapper_type=LLMFrameworkEnum.AUTOGEN)
+
+        # Make a call that should fail
+        messages = [UserMessage(content="Hello", source="user")]
+
+        with pytest.raises(Exception):  # Should raise authentication or API error
```

Ruff (0.14.10)

Do not assert blind exception: `Exception`

(B017)